### PR TITLE
feat(css/parser): Added `OperatorValue` node

### PR DESCRIPTION
**Pull request by alexander-akait**
Added `OperatorValue` AST Node, it represents any `delim` tokens.

For example:
```css
a {
   font: italic small-caps bold 12px/30px Georgia, serif;
}
```

is valid CSS, but `/` should be not consider as division. `/` should be consider as `division` only for math functions like `calc()`/`min()`/`max()`/etc, we will improve them late.

The only thing I doubt is the name. 
Alternative - `DelimiterValue`.
Note - this node will be used for `,` in future.

### CODE REVIEW

**Comment by alexander-akait** [comment]
@kdy1 Can you all and give your opinion?

**Comment by kdy1** [comment]
Is this a groundwork for 

> / should be not consider as division. / should be consider as division only for math functions like calc()/min()/max()

?

I think I didn't get the point of this kind of node. Is it defined on the css specification?

**Comment by alexander-akait** [comment]
@kdy1 

> I think I didn't get the point of this kind of node. Is it defined on the css specification?

In CSS spec no division, `/` can have many meanings, in

```css
a {
  font: italic small-caps bold 12px/30px Georgia, serif;
}
```

it is just separator in declaration value.

In:

```css
div {
  width: calc(100px / 2);
}
```

it is `/`` division.

Generally in CSS spec we have `<delim-token>`:

> <delim-token> has a value composed of a single code point.

So `*`, `+`, `-` and other not reserved characters (and even `,` (comma)) are delim characters.

Therefore, we need AST node in order to present it in AST. `OperatorValue` do it, but the name worries me a little, it can be misleading, so I suggest to use `DelimiterValue` and want to heart your opinion here.

**Comment by alexander-akait** [comment]
@kdy1 Friendly ping

File: css/parser/tests/recovery/delim-token/ampersand/output.swc-stderr [hunk 1]
```diff
@@ -1,6 +0,0 @@
-error: Expected Declaration value
```
Comment (kdy1) [inline css/parser/tests/recovery/delim-token/ampersand/output.swc-stderr:1]: This test should have error message, as it's in `recovery` test.

File: css/parser/tests/recovery/delim-token/asterisk/output.json [hunk 2]
```diff
@@ -64,59 +64,34 @@
         },
         "items": [
           {
-            "type": "Tokens",
```
Comment (kdy1) [inline css/parser/tests/recovery/delim-token/asterisk/output.json:67]: Is this change intentional?

File: css/parser/tests/recovery/delim-token/asterisk/output.swc-stderr [hunk 3]
```diff
@@ -1,6 +0,0 @@
-error: Expected Declaration value
```
Comment (kdy1) [inline css/parser/tests/recovery/delim-token/asterisk/output.swc-stderr:1]: Same here, it should report an error.

File: css/parser/tests/recovery/delim-token/bar/output.swc-stderr [hunk 4]
```diff
@@ -1,6 +0,0 @@
-error: Expected Declaration value
```
Comment (kdy1) [inline css/parser/tests/recovery/delim-token/bar/output.swc-stderr:1]: Same here.

**Review by kdy1 — CHANGES_REQUESTED** [review]
I found some issues with updated test references, but I added review only on first few files as other cases are similar to them.


And it seems like tests are broken. Can you rebase it?

File: css/parser/tests/recovery/delim-token/bar/output.swc-stderr (hunk 4 above)
Comment (alexander-akait) [reply css/parser/tests/recovery/delim-token/bar/output.swc-stderr:1]: Yes, it is valid output now, we need to move them in other place

**Comment by alexander-akait** [comment]
@kdy1 I would also like to know what you think about the `OperatorValue` name

**Comment by kdy1** [comment]
Oh, I prefer `OpValue` over `OperatorValue`. Thanks! I missed it.

**Comment by alexander-akait** [comment]
Close in favor https://github.com/swc-project/swc/pull/3402
